src/bio/SConscript

```diff
@@ -18,15 +18,13 @@ def scons():
     denv.AppendUnique(OBJPREFIX='b_')
 
     # SPDK related libs
-    libs = ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'spdk_copy']
-    libs += ['rte_mempool', 'rte_mempool_ring', 'rte_bus_pci']
-    libs += ['rte_pci', 'rte_ring', 'rte_mbuf', 'rte_eal', 'rte_kvargs']
-    libs += ['spdk_bdev_aio', 'spdk_bdev_nvme', 'spdk_bdev_malloc']
-    libs += ['spdk_conf', 'spdk_blob', 'spdk_nvme', 'spdk_util']
+    libs = ['spdk_env_dpdk', 'spdk_thread', 'spdk_bdev', 'rte_mempool']
+    libs += ['rte_mempool_ring', 'rte_bus_pci', 'rte_pci', 'rte_ring']
+    libs += ['rte_mbuf', 'rte_eal', 'rte_kvargs', 'rte_power', 'spdk_bdev_aio']
+    libs += ['spdk_bdev_nvme', 'spdk_blob', 'spdk_nvme', 'spdk_util']
     libs += ['spdk_json', 'spdk_jsonrpc', 'spdk_rpc', 'spdk_trace']
     libs += ['spdk_sock', 'spdk_log', 'spdk_notify', 'spdk_blob_bdev']
-    libs += ['spdk_vmd', 'spdk_event', 'spdk_event_bdev', 'spdk_event_copy']
-    libs += ['spdk_event_vmd']
+    libs += ['spdk_vmd', 'spdk_event', 'spdk_event_bdev', 'spdk_init']
```

Comment:
I'm not sure why the event libs are still required, asked Jim. Let's keep them for this moment.

Reply:
answer is that spdk_event is not required and spdk_event_bdev is actually not a related event lib and is confusingly named, it will be renamed to spdk_init_bdev probably and is required by bio